src/utils/deflate.js

```diff
@@ -22,12 +22,12 @@ async function browserDeflate(buffer) {
 function testCompressionStream() {
   try {
     const cs = new CompressionStream('deflate')
+    cs.writable.close()
     // Test if `Blob.stream` is present. React Native does not have the `stream` method
     const stream = new Blob([]).stream()
     stream.cancel()
-    return !!cs
+    return true
   } catch (_) {
-    // no bother
+    return false
```

Comment:
return false is already handled in next lines

Reply:
I removed return false from outside the try catch block since the only way false is returned is if an error is thrown.

An empty catch block is frowned upon by linters, which is presumably why the comment was put there, to silence linter errors.

Reply:
Ah, didn't notice that the return was removed.